qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -32,10 +32,14 @@
     RegisterSlot,
     MemorySlot,
     Schedule,
+    InstructionScheduleMap,
 )
 from qiskit.pulse.channels import PulseChannel
 from qiskit.circuit import Parameter, ParameterExpression
+from qiskit.providers.backend import BackendV1 as Backend
```

Comment:
Since terra 0.19 has been released this could be also BackendV2. We need to add support for this in follow-up PR.

Reply:
Yes, in 0.3.